handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -157,6 +158,10 @@ public Boolean run() {
                             }
                         } catch (Throwable ignore) {
                             logger.debug("KeyManagerFactory not supported.");
+                        } finally {
+                            if (cert != null) {
+                                cert.delete();
```

Comment:
Looks good if it's safe to delete the file before free'ing the pointers in the outer `try-finally`.